cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -234,6 +202,44 @@ script "js_filter_dimensions", type: "javascript" do
   EOS
 end
 
+datasource "billing_centers" do
+  request do
+    auth $auth_flexera
+    host rs_optima_host
+    path join(["/analytics/orgs/",rs_org_id,"/billing_centers"])
+    header "Api-Version", "1.0"
+    header "User-Agent", "RS Policies"
+    query "view", "allocation_table"
+  end
+  result do
+    encoding "json"
+    collect jmes_path(response,"[*]") do
+      field "href", jmes_path(col_item,"href")
+      field "id", jmes_path(col_item,"id")
+      field "name", jmes_path(col_item,"name")
+      field "parent_id", jmes_path(col_item,"parent_id")
+      field "ancestor_ids", jmes_path(col_item,"ancestor_ids")
+      field "allocation_table", jmes_path(col_item,"allocation_table")
+    end
+  end
+end
+
+datasource "ds_top_level_billing_centers" do
+  run_script $js_top_level_bc, $ds_listed_billing_centers
+end
+
+script "js_top_level_bc", type: "javascript" do
```

Comment:
I think all scripts should be after all data sources.

Reply:
Hi Juan regarding this article; Datasources and Scripts now are presented in the order of execution followed by any script blocks that they call.
https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices